test/parallel/test-cluster-debug-port.js

```diff
@@ -1,42 +1,72 @@
 'use strict';
-require('../common');
+
+// This is hack to make master process think it is running with --debug
+if (!process.env.forked) {
```

Comment:
Can you spawn this file so that the `execArgv` will be set automatically and we can properly test this? For eg, like in test/parallel/test-cluster-bind-twice.js.